manifests/charts/istio-telemetry/mixer-telemetry/values.yaml

```diff
@@ -101,3 +101,190 @@ mixer:
     podAntiAffinityTermLabelSelector: []
 
 revision: ""
+
+global:
+  # Specify pod scheduling arch(amd64, ppc64le, s390x) and weight as follows:
+  #   0 - Never scheduled
+  #   1 - Least preferred
+  #   2 - No preference
+  #   3 - Most preferred
+  arch:
+    amd64: 2
+    s390x: 2
+    ppc64le: 2
+
+  # set the default set of namespaces to which services, service entries, virtual services, destination
+  # rules should be exported to. Currently only one value can be provided in this list. This value
+  # should be one of the following two options:
+  # * implies these objects are visible to all namespaces, enabling any sidecar to talk to any other sidecar.
+  # . implies these objects are visible to only to sidecars in the same namespace, or if imported as a Sidecar.egress.host
+  defaultConfigVisibilitySettings: []
+#  - '*'
```

Comment:
stray line?

Reply:
It was copied over https://github.com/istio/istio/blob/master/manifests/charts/global.yaml#L307 but I'll remove it